pkg/devserver/discovery/discovery.go

```diff
@@ -71,7 +73,7 @@ func Autodiscover(ctx context.Context) map[string]struct{} {
 		for _, path := range Paths {
 			// These requests _should_ be fast as we know a port is open,
 			// so we do these sequentially.
-			url := fmt.Sprintf("http://127.0.0.1:%d%s", port, path)
+			url := util.NormalizeAppURL(fmt.Sprintf("http://127.0.0.1:%d%s", port, path), false)
```

Comment:
This fixes an issue where, even if polling was disabled, an app would be discovered at this URL and pinged, as this raw URL was different to the normalized URL within the existing app.